lib/Aftok/Payments.hs

```diff
@@ -51,6 +57,7 @@ import Aftok.Payments.Types
   )
 import qualified Aftok.Payments.Types as PT
 import qualified Aftok.Payments.Zcash as Zcash
+import qualified Aftok.Payments.Zcash.Types as Zcash
```

Comment:
Does this alias the previous line's import?

Reply:
No! This is one of the really nice things about qualified imports, is that you get the union of the two namespaces under a single prefix.

Reply:
If there is a conflict (e.g. Foo in both packages) does that result in a compilation failure or an alias?

Reply:
Compilation failure.